### PR TITLE
Updated publishing to SAR

### PR DESCRIPTION
**Pull request by svenngjeto**

### CODE REVIEW

File: buildspec_publish_to_sar.yml [hunk 1]
```diff
@@ -6,17 +6,21 @@ phases:
...
     commands:
       # Use AWS SAM to build and package the application by using AWS CloudFormation
-      - sam build  --debug
-      - sam package --debug --template-file .aws-sam/build/template.yaml --s3-bucket $S3_BUCKET --output-template-file sampackaged.yaml --force-upload
+      - sam build
+      - sam package --template-file .aws-sam/build/template.yaml --s3-bucket $S3_BUCKET --output-template-file sampackaged_raw.yaml
   post_build:
     commands:
+      # Getting latest version tag from git
+      - APPLICATION_VERSION=$(curl -s  https://api.github.com/repos/BIBSYSDEV/$GIT_REPO/releases/latest | jq -r '.tag_name')
```
Comment (brinxmat) [inline buildspec_publish_to_sar.yml:19]: What happens when tags are not present?

**Review by brinxmat — APPROVED** [review]

File: template.yaml [hunk 2]
```diff
@@ -10,9 +10,9 @@ Metadata:
     Name: NvaDoiTransformer
     Description: Backend for transforming response from DOI source to internal NVA data model
     Author: Unit
-    SemanticVersion: 1.0.0
     SpdxLicenseId: MIT
     LicenseUrl: LICENSE
+    Labels: ['${CODEBUILD_RESOLVED_SOURCE_VERSION}', '${GIT_REPO}']
```
Comment (janbj) [inline template.yaml:15]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Found an embedded parameter outside of an &#34;Fn::Sub&#34; at Metadata/AWS::ServerlessRepo::Application/Labels/1

Comment (janbj) [inline template.yaml:15]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Found an embedded parameter outside of an &#34;Fn::Sub&#34; at Metadata/AWS::ServerlessRepo::Application/Labels/0

File: buildspec_publish_to_sar.yml (hunk 1 above)
Comment (svenngjeto) [reply buildspec_publish_to_sar.yml:19]: Pipeline fails... I belive that is what we want.... But the webhook triggers only on release event

File: template.yaml (hunk 2 above)
Comment (svenngjeto) [reply template.yaml:15]: cfn-lint warns about the same, but we dont care. Is part of envsubst and labelling the artifact in SAR

Comment (svenngjeto) [reply template.yaml:15]: cfn-lint warns about the same, but we dont care. Is part of envsubst and labelling the artifact in SAR

**Pull request merged**